src/Symfony/Component/Validator/Validator/RecursiveContextualValidator.php

```diff
@@ -305,6 +305,10 @@ protected function normalizeGroups($groups)
     private function validateObject($object, string $propertyPath, array $groups, int $traversalStrategy, ExecutionContextInterface $context)
     {
         try {
+            if (!\is_object($object)) {
+                throw new NoSuchMetadataException(sprintf('Cannot create metadata for non-objects. Got: "%s".', get_debug_type($object)));
```

Comment:
`sprintf('Cannot create metadata for non-objects, "%s" given.', \get_class($object))`

(we use get_debug_type on 5.2+)

Reply:
Ah, for once I'm too modern :D Will fix